dataset_test.go

```diff
@@ -1,66 +0,0 @@
-package ld
```

Comment:
This file was accidentally deleted. BTW when you resolve the conflicts, ensure that it prints exactly 819 (I could miss updating 766 to 819).